thrust/detail/type_traits.h

```diff
@@ -160,11 +160,11 @@ template<typename T> struct has_trivial_copy_constructor
       is_pod<T>::value
 #if THRUST_HOST_COMPILER == THRUST_HOST_COMPILER_MSVC || \
     THRUST_HOST_COMPILER == THRUST_HOST_COMPILER_CLANG
-      || __has_trivial_copy(T)
+      || __is_trivially_constructible(T)
```

Comment:
The [clang language extensions page](https://clang.llvm.org/docs/LanguageExtensions.html) says
> `__has_trivial_copy` (GNU, Microsoft): Deprecated, use `__is_trivially_copyable` instead.

Reply:
Sorry for the delay, I've changed it as per your comment.